client/src/modules/report-group/index.html

```diff
@@ -0,0 +1,170 @@
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.ADMIN" | translate }}</li>
+      <li class="static">{{ "Reporting Group" | translate }}</li>
+    </ol>
+
+    <div class="toolbar">
+      <div class="toolbar-item">
+        <a  class="btn btn-default pull-right"
+          ui-sref="emailreport"
+          ui-sref-opts="{notify:false}">
+            <i class="fa fa-plus"></i>
+            <span  translate=""  class="ng-scope">
+              Email Report
+            </span>
+        </a>
+      </div>
+
+    </div>
+  </div>
+</div>
+
+
+
+<div class="flex-content">
+  <div class="container-fluid" id="anchor">
+
+  <div class="col-lg-12">
+    <!-- start new Reporting group form -->
+    <div class="col-lg-5">
+    <h4> {{"New reporting group"}}</h4>
```

Comment:
The correct way to write this in is using the `translate` directive.  First, you must make a key in the `i18n` JSONs.  Try this:
 1. Create a file called `report-group.json` in the folder `client/src/i18n/en/`.
 2.  Put the following code inside it:
  ```json
{
  "REPORT" : {
    "NEW_GROUP" : "New Reporting Group"
   }
}
```
 3. Do the same thing in a new file, `report-group.json` in the folder `client/src/i18n/fr/`, but with french translations.
 4. Change the above HTML to be:
```html
<h4 translate>REPORT.NEW_GROUP</h4>
```

You have completed a translation in BHIMA!

If you need any help or clarity, anyone on the local team can help you out.  We are all very good at this now :dancer: